scripts/bootstrap-ansible.sh

```diff
@@ -71,6 +71,10 @@ pushd ${OA_DIR}
     exit 99
   fi
 
+  # Run the RPC-MaaS "GET" playbook to clone RPC-MaaS into place.
+  ansible-playbook -i 'localhost 127.0.0.1,' \
```

Comment:
ansible-playbook -i '127.0.0.1,'  or ansible-playbook -i 'localhost,'  should be enough.